pkgs/development/python-modules/openshift/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib
+  , buildPythonPackage
+  , fetchPypi
+  , jinja2
+  , kubernetes
+  , ruamel-yaml
+  , six
+  , python-string-utils
```

Comment:
This formatting is a bit non-standard. I wouldn't usually call this out, but this doesn't fit any of the patterns I've seen used commonly.